prow/plugins/branchcleaner/README.md

```diff
@@ -0,0 +1,14 @@
+# Branchcleaner
+
+The `branchcleaner` makes Prow automatically delete the source branch of a Pull Request if it is
+on the same repo as the target branch. This is helpful to keep repos that don't allow forking clean.
+
+## Usage
+
+Enable the `branchcleaner` in the desired repos via the `plugins.yaml`:
+
+```
+plugins:
+  my-github/repo:
```

Comment:
nit: can we rename this as `org/repo` - feels more intuitive and consistent with naming in other places in test-infra. :grimacing:

Reply:
Done